README.md

```diff
@@ -1,92 +1,150 @@
-:hammer_and_wrench: Strawberry - Build with Visual Studio 2022
-================================================================
+# :hammer_and_wrench: Strawberry - Build with Visual Studio 2022
 
-This guide uses Visual Studio 2022 to build Strawberry as well all required libraries.
+This guide uses Visual Studio 2022 to build Strawberry Music Player as well as required dependencies.
 
 These instructions are provided as-is, they are primarily intended for developers working on Strawberry on Windows.
 
 We do not offer support to users for building Strawberry on Windows.
 
 Build tools (Git, CMake, Meson, Perl, Python, etc) versions are not regulary updated, so make sure you bump the versions in versions.bat before you run install.bat.
 
-### Requirements
```

Comment:
I'd like to keep the requirements list.

Reply:
Restored in 5d749b413e845a23772b20f69a6b5b8aa638ae6f.